OrbitCore/FunctionUtils.cpp

```diff
@@ -0,0 +1,179 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "FunctionUtils.h"
+
+#include <map>
+
+#include "Capture.h"
+#include "Log.h"
+#include "OrbitBase/Logging.h"
+#include "Utils.h"
+#include "cvconst.h"
+
+#ifdef _WIN32
+#include "SymbolUtils.h"
+#endif
+
+namespace function {
```

Comment:
We haven't really made up our minds around namespaces yet, but this seems a bit too generic. Maybe `FunctionUtils` like the file?

Reply:
You're right, and as I'm going to change several more classes in the same way we can agree how to name namespaces to follow this logic in the next PRs. Alternatively, we can use prefix orbit (`orbit::function`) to make it clear that namespace works with Orbit-specific classes.